src/mbgl/gl/segment.hpp

```diff
@@ -11,23 +11,39 @@
 namespace mbgl {
 namespace gl {
 
-template <class Attributes>
-class Segment {
+class SegmentInfo {
```

Comment:
That's what I attempted initially as well, but I opted to do it the other way around to keep the diff size small. However, I'm happy to rename `Segment` to `DrawableSegment`. Another candidate might be `SegmentBuffers` for naming alignment with `VertexBuffer` and `IndexBuffer`.